src/gennylib/src/PoolManager.cpp

```diff
@@ -60,7 +60,11 @@ mongocxx::pool::entry genny::v1::PoolManager::client(const std::string& name,
     std::unique_lock<std::mutex> lock{lap.first};
 
     Pools& pools = lap.second;
-
+    auto perInstance =
```

Comment:
Is there something missing here? It doesn't look like you use perInstance anywhere.

Reply:
I assume you mean the yml config (as the variable is used on the next line).

The configuration is demonstrated [here](https://github.com/mongodb/genny/pull/531/files#diff-dc702b545e53500c48ea324dd47ba077c92d81a33320be2bebdbdeb8cbc2a394R48).

Reply:
What I was missing was that support for multiple instances has existed for 3 years in the code base and never used  :) I just spent some time in git blame and it makes more sense to me now.